service/src/main/resources/application.yml

```diff
@@ -10,6 +10,8 @@ management:
         include: info, health, metrics, mappings
 
 gp2gp:
+  trustStoreUrl: ${GP2GP_SSL_TRUST_STORE_URL:}
+  trustStorePassword: ${GP2GP_SSL_TRUST_STORE_PASSWORD:}
```

Comment:
for gpc we have certificates stored under there, i think it would be most consistent with this adapter to have it under storage

Reply:
Ok